public/style.css

```diff
@@ -0,0 +1,143 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  /* width: 100%; */
+  height: 100%;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #f7fbfb;
+}
+
+header {
+  background-color: midnightblue;
+  color: white;
+  margin-bottom: 2.5rem;
+  padding: 1.5rem 0;
+  text-align: center;
+  box-shadow: 0 2px 4px 0 rgba(14, 30, 37, 0.5);
+}
+
+section {
+  padding: 1.5rem;
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(14, 30, 37, 0.12);
+  border-radius: 8px;
+}
+
+h1 {
+  font-size: 1rem;
+}
+
+.app {
+  padding: 0 1.5rem 1.5rem 1.5rem;
+  background-color: transparent;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 1rem;
+  justify-content: center;
+}
+
+.list-description {
+  margin: 0rem 0 2rem 0;
+}
+
+ul {
+  margin: 0rem;
+  list-style-type: none;
+}
+
+li {
+  margin-bottom: 1rem;
+}
+
+li:last-of-type {
+  margin-bottom: 0;
+}
+
+.endpoint-description {
+  font-weight: bold;
+  margin-bottom: 0.3rem;
+  background-color: ;
+}
+
+.endpoint-url {
+  display: inline-block;
+  font-style: italic;
+  color: blue;
+  padding: 0.2rem 0.5rem;
+  text-decoration: none;
+  transition: all 300ms ease-in-out;
+}
+
+.endpoint-url:hover {
+  cursor: pointer;
+  background: rgba(255, 255, 0, 1);
+}
+
+.endpoint-url:visited {
+  color: blue;
+}
+
+.endpoint {
+  /* text-decoration: underline;
+  text-underline-position: below;
+  text-underline-offset: 5px;
+  text-decoration-thickness: 2px;
+  text-decoration-color: greenyellow;
+  -webkit-text-decoration-color: greenyellow;
+  -moz-text-decoration-color: greenyellow; */
+  padding: 0.1rem 0rem;
+  border-bottom: 2px solid greenyellow;
+}
+
+.object {
+  margin: 1rem 0;
+}
+
+.object p {
+  font-weight: bold;
+  font-size: 0.8rem;
+}
+
+.correct p {
+  color: green;
+}
+
+.incorrect p {
+  color: red;
+}
+
+.incorrect pre code span {
+  color: red;
+}
+
+pre {
+  display: flex;
+  justify-content: flex-start;
+  margin: 0.4rem 0;
+}
+
+code {
+  display: block;
+  width: 100%;
+  border-radius: 4px;
+  padding: 0.6rem;
+  background-color: rgba(0, 0, 0, 0.05);
+}
+
+@media screen and (min-width: 960px) {
+  h1 {
+    font-size: 2rem;
+  }
+}
```

Comment:
I thought the presentation was very nice.  From a usability perspective, I found myself wanting to clear the filter even when there were results.

Another idea is to somehow include information about why this result matched the filter...

Reply:
You are right about that. I just ran out of time when I was doing the final touches before your code review. But there should definitively be a clear filter button on the front page.

And your idea about presenting what filter options that are selected for a given number of item, is another great idea. 

Another thing that would be cool (i've seen it on other sites) is to prefetch the number of shows that your current query will show when the filter is applied. It would be like a prefetch of the query, but just with the number of shows for that query, if you know what I mean.